util/dvsim/nc_post_cmd.sh

```diff
@@ -0,0 +1,21 @@
+#! /bin/sh
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+#ignore error 6, it is due to some misconfig of nc commands
```

Comment:
Can you expand this comment a bit further? Do we see commands that run successfully to completion but return with a status of 6 (SIGABRT)?

Reply:
there was some issue with the nc farm setup earlier that needed this workaround. it has since been resolved and this workaround is no longer needed